tests/forms-schema/forms-schema.test.ts

```diff
@@ -7787,6 +7787,35 @@ describe('external id generation and personalisation', () => {
     ).toThrow('"personalisation.type" must be one of [CALLBACK, ONEBLINK_API]')
   })
 
+  test('should throw error when attempting to save a "RECEIPT_ID" with 2 "sequentialNumber" components', () => {
+    const configuration = {
+      receiptComponents: [
+        {
+          type: 'sequentialNumber',
+        },
+        {
+          type: 'text',
+          value: 'value',
+        },
+        {
+          type: 'sequentialNumber',
+        },
+      ],
+    }
+
+    expect(() =>
+      validateFormThrowError({
+        ...form,
+        externalIdGenerationOnSubmit: {
+          type: 'RECEIPT_ID',
+          configuration,
+        },
+      }),
+    ).toThrow(
+      'Receipt Components can only contain one "Sequential Numbers" type.',
+    )
+  })
```

Comment:
Can we also please write a test that passes if there is only one (if this test does not exist already)